src/odemis/gui/cont/views.py

```diff
@@ -373,6 +373,21 @@ def _initialize_ovv_im(self, shape):
         ovv_im.metadata[MD_POS] = self.m_view.view_pos.value
         return ovv_im, mpp
 
+    def reset_ovv(self):
+        """
+        Reset the overview image and history after a new sample has been loaded
+        """
+        md = self.ovv_im.metadata
+        self.ovv_im = DataArray(numpy.zeros(OVV_SHAPE, dtype=numpy.uint8))
+        self.ovv_im.metadata = md
+        self.im_opt = copy.deepcopy(self.ovv_im)
+        self.im_sem = copy.deepcopy(self.ovv_im)
+
+        self._data_model.stage_history.value = []
```

Comment:
Replace this line (which resets the stage_history) by the current content of the if block in `_on_chamber_state()`. => same thing but also a comment and keep the latest position.